### PR TITLE
oneccl rfc process

### PR DESCRIPTION
**Pull request by rscohn2**
This is the onednn process with minimal changes.

@vpirogov

### CODE REVIEW

**Review by garzaran — APPROVED** [review]
THis looks good

**Comment by rscohn2** [comment]
Merging based on Maria's approval. I can make changes post-merge if you have comments.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# oneCCL Design Documents / RFCs
+
+The purpose of the RFC process is to communicate the intent to make
+library-wide changes, get the feedback prior to the actual implementation,
+increase the transparency on why and how the decisions are made, and improve
+the alignment between different teams involved in oneDNN development.
```
Comment (nikitaxgusev) [inline README.md:6]: Is this typo `oneDNN development`?

Comment (rscohn2) [reply README.md:6]: Fixed in #136 . Thanks

**Pull request merged**